src/uct/base/uct_iface.c

```diff
@@ -280,6 +280,24 @@ int uct_iface_is_reachable_v2(const uct_iface_h tl_iface,
     return iface->internal_ops->iface_is_reachable_v2(tl_iface, params);
 }
 
+int uct_ep_is_conntected(const uct_ep_h ep,
+                         const uct_ep_is_connected_params_t *params)
+{
+    const uct_base_iface_t *iface;
+
+    if (!ucs_test_all_flags(params->field_mask,
+                            UCT_EP_IS_CONNECTED_FIELD_DEVICE_ADDR |
+                            UCT_EP_IS_CONNECTED_FIELD_IFACE_ADDR |
+                            UCT_EP_IS_CONNECTED_FIELD_EP_ADDR)) {
+        ucs_error("missing params (field_mask: %lu), device_addr, iface_addr "
+                  " and ep_addr should be supplied.", params->field_mask);
```

Comment:
`should` -> `must` (actually that's wrong in the other place as well, but don't fix there now).
Maybe `supplied` -> `provided`